Show matched emoji name instead of an ID in emoji suggestion box (#2656)
ckeditor/ckeditor4: tests/plugins/emoji/manual/font.md

@bender-tags: 4.14.0, feature, emoji, 2583
@bender-ckeditor-plugins: wysiwygarea, toolbar, emoji
@bender-ui: collapsed
@bender-include: ../_helpers/tools.js

> 1. Focus editor.

Reviewer: ```suggestion
1. Focus the editor.
```

> 1. Type `:smiling_face:`

Reviewer: ```suggestion
2. Type `:smiling_face:` in the editor's content.
```

Reviewer 2: See https://github.com/ckeditor/ckeditor4/pull/2656#discussion_r347772593

## Expected:

> Nice colored emoji is displayed in emoji suggestion box: <span style="font-family:&quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;;">☺</span>

Reviewer: ```suggestion
Emoji suggestion box displays a nicely colored emoji symbol like this one: <span style="font-family:&quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;, &quot;Segoe UI Symbol&quot;;">☺</span>.
```

## Unexpected:

> Ugly black and white smiley is displayed in emoji suggestion box: <span style="font-family:sans-serif">☺</span>

Reviewer: ```suggestion
Emoji suggestion box displays an ugly black and white smiley like this one: <span style="font-family:sans-serif">☺</span>
```
